docs/intro-Cloud/flavors.md

```diff
@@ -57,7 +57,7 @@ machine (VM) instance like system memory size (RAM), virtual cores
 
 The flavor's root disk size is the amount of disk space used by the root
 (*/*) partition, an ephemeral disk that the base image is copied into
-(see section [launch an instance](./ch_LaunchInstances.md#launch-an-instance)
+(see section [launch an instance](#launch-an-instance)
```

Comment:
Hmm, this probably doesn't produce a functional link, since that section is in `launch_instance.md`?